src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/sentryApplicationRow.jsx

```diff
@@ -111,7 +112,22 @@ export default class SentryApplicationRow extends React.PureComponent {
           ) : (
             <Box>
               {app.status === 'unpublished' ? (
-                this.renderRemoveApp(app)
+                <Access access={['org:admin']}>
+                  {({hasAccess}) => (
+                    <React.Fragment>
+                      {!hasAccess && (
+                        <Tooltip
+                          title={t('Owner permissions are required for this action.')}
+                        >
+                          <span>
```

Comment:
Do you need this span? I thought @EvanPurkhiser or @billyvg did changes recently to remove the need for the span.

Reply:
We can get rid of these now! Yes!